integration/spark/src/main/scala/org/apache/spark/sql/execution/command/management/RefreshCarbonTableCommand.scala

```diff
@@ -83,6 +84,23 @@ case class RefreshCarbonTableCommand(
       if (FileFactory.isFileExist(schemaFilePath)) {
         // read TableInfo
         val tableInfo = SchemaReader.getTableInfo(identifier)
+        val tableProperties = tableInfo.getFactTable.getTableProperties
+        val indexName = tableProperties.get(CarbonCommonConstants.SPATIAL_INDEX)
+        if (indexName != null) {
+          val SPATIAL_INDEX_CLASS = s"${ CarbonCommonConstants.SPATIAL_INDEX }.$indexName.class"
```

Comment:
can create new variable for CarbonCommonConstants.SPATIAL_INDEX and format the code

Reply:
Done